core/config/locales/en.yml

```diff
@@ -1623,7 +1625,6 @@ en:
       customer_support_email_help: "This email is visible to your Store visitors in the Footer section"
       new_order_notifications_email_help: "If you want to receive an email notification every time someone places an Order please provide an email address for that notification to be sent to"
       seo_robots: "Please check <a href='https://developers.google.com/search/reference/robots_meta_tag' target='_blank'>this page for more help</a>"
-    store_set_default_button: Set as default
```

Comment:
@MatthewKennedy shouldn't this be kept?

Reply:
Thats a duplicate key found:

<img width="611" alt="Screenshot 2021-04-03 at 14 06 04" src="https://user-images.githubusercontent.com/1240766/113479237-c9ac3700-9485-11eb-8a05-62590afc88da.png">

Reply:
YML linter was telling me duplicate key.